app/src/main/java/de/grobox/transportr/networks/Region.java

```diff
@@ -56,6 +56,7 @@
 	CANADA(R.string.np_region_canada, "🇨🇦"),
 	COSTA_RICA(R.string.np_region_costa_rica, "🇨🇷"),
 	AFRICA(R.string.np_region_africa, "🌍");
+	NICARAGUA(R.string.np_region_nicaragua, "🌍");
```

Comment:
Here, you should add the flag of Nicaragua as a second parameter.

Reply:
It's the UTF8-Representation of the flag, see https://emojipedia.org/flag-for-nicaragua/